department-of-veterans-affairs/va.gov-cms
# VAGOV-3311: Replace moderation dashboard with new sections listing.

QA: 

As anon
[ ] Homepage should look and behave the same as before

As a content_editor (or content_reviewer or content_publisher)
[ ] login to the site. You should be taken to /sections instead of the moderation dashboard
[ ] On /sections, you should see the same hierarchy that's visible through the Sections menu in the toolbar. 
[ ] You should no longer be able to access moderation dashboard 
[ ] You should see new options in the Content menu, including Files
[ ] You should be able to go to /section/vha and break content locks from there
[ ] On /section/vha, you the child sections should now have bullets, whereas before they did not.

As admin
[ ] Note that "rebuild trigger" menu item has changed to "Rebuild front end"
[ ] Note that Panels, Moderation Dashboard, Page Manager, Better Field Descriptions, and Content Model UI are no longer enabled.
